### PR TITLE
Add VS Code to Docs

### PR DESCRIPTION
**Pull request by leoebfolsom**

### CODE REVIEW

File: docs/guides/vs_code_extension.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+---
+id: vs_code_extension
+title: VS Code
```
Comment (sungchun12) [inline docs/guides/vs_code_extension.md:3]:
```suggestion
title: VS Code Extension
```

File: docs/guides/vs_code_extension.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+---
+
+With the Datafold VS Code IDE, you can quickly run and diff any dbt models, view and interact with diff results in a clean GUI, and automatically diff models that change all within your VS Code IDE.
```
Comment (sungchun12) [inline docs/guides/vs_code_extension.md:6]:
```suggestion
With the Datafold VS Code Extension, you can quickly run and diff any dbt models, view and interact with diff results in a clean GUI, and automatically diff models that change all within your VS Code IDE.
```

**Review by sungchun12 — CHANGES_REQUESTED** [review]
Quick changes. Then you can squash and merge

**Comment by leoebfolsom** [comment]
fyi @KiraFuruichi

File: docs/guides/vs_code_extension.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+![](../../static/img/vs_code_datafold_gif.gif)
+
+## Installation
+
+1. Install the Extension using the VS Code Extension tab.
+
+![](../../static/img/vs_code_install.png)
+
+2. Open up the settings to set file paths for dbt profiles, your dbt project, and your Python installation.
```
Comment (KiraFuruichi) [inline docs/guides/vs_code_extension.md:16]:
```suggestion
2. Open up the extension's settings to set file paths for dbt profiles, your dbt project, and your Python installation.
```

File: docs/guides/vs_code_extension.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+This is what will be added to the `vars` of your `dbt_project.yml` once setup is complete:
+
+```
+vars:
+  data_diff:
+    prod_database: YOUR_DATABASE
+    prod_schema: YOUR_SCHEMA
+```
+
+## Getting Started
```
Comment (KiraFuruichi) [inline docs/guides/vs_code_extension.md:47]:
```suggestion
## Getting started
```

File: docs/guides/vs_code_extension.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+:::tip
+
+In order for the data diff to run, there must be a version of the model in both your production and development environments.
+
+:::
+
+![](../../static/img/vs_code_first_diff.png)
+
+### Explore Value-Level Differences
```
Comment (KiraFuruichi) [inline docs/guides/vs_code_extension.md:65]:
```suggestion
### Explore value-level differences
```

File: docs/guides/vs_code_extension.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+Understand exactly which values will change if the development code is merged to production.
+
+![](../../static/img/vs_code_diff_values_trial.png)
+
+### Additional Commands
```
Comment (KiraFuruichi) [inline docs/guides/vs_code_extension.md:71]:
```suggestion
### Additional commands
```

File: docs/guides/vs_code_extension.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+![](../../static/img/vs_code_command_diff_dbt_model.png)
+
+![](../../static/img/vs_code_command_diff_dbt_model_select.png)
+
+You can also diff every model that was built in your most recent `dbt run`.
+
+![](../../static/img/vs_code_command_diff_dbt_run.png)
+
+
+## Additional Features
```
Comment (KiraFuruichi) [inline docs/guides/vs_code_extension.md:93]:
```suggestion
## Additional features
```

**Review by KiraFuruichi — COMMENTED** [review]
minor suggestions

**Pull request merged**